### PR TITLE
Math support files

### PR DESCRIPTION
**Pull request by jonludlam**
Math support currently doesn't work because `odoc support-files` doesn't actually write the vendored katex files. This PR fixes that.

Note that dune doesn't quite work yet with this as it's only expecting the original 2 files to be output, and while it will work with the first build, rebuilds don't work as expected. This is related to https://github.com/ocaml/dune/issues/1556.

### CODE REVIEW

File: src/vendor/dune [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  (:x odoc_vendor.ml)
+  (glob_files *.js)
+  (glob_files *.css)
+  (glob_files_rec *.woff2))
+ (action
+  (progn
+   (setenv
+    SOURCE_DATE_EPOCH
+    0
+    (run ocaml-crunch . -e js -e css -e woff2 -o %{x}.corrected -m plain))
```
Comment (Julow) [inline src/vendor/dune:20]: This prints something, which is not useful to know given how fast it runs.
```suggestion
    (run ocaml-crunch . -e js -e css -e woff2 -o %{x}.corrected -m plain -s))
```

**Review by Julow — APPROVED** [review]
Looks good to me :)

File: src/vendor/dune (hunk 1 above)
Comment (jonludlam) [reply src/vendor/dune]: Just tried this, but `-s` doesn't work on older crunches, so I'll revert.

Comment (jonludlam) [reply src/vendor/dune]: Well older crunches produce different output anyway, so that's not much help.

**Pull request merged**